content/en/blog/_posts/2017-08-00-High-Performance-Networking-With-Ec2.md

```diff
@@ -73,8 +73,5 @@ When using Romana v2.0, native VPC networking is now available for clusters of a
 ![](https://archive.org/download/hpc-ec2-vpc-2/hpc-ec2-vpc-2.png)
 
 
-The preview release of Romana v2.0 is available [here](http://romana.io/preview). We welcome comments and feedback so we can make EC2 deployments of Kubernetes as fast and reliable as possible.    
```

Comment:
I'd prefer not to remove this whole paragraph. Could we do something like this instead:

```suggestion
The preview release of Romana v2.0 is available here _Editor's Note:_ link removed. We welcome comments and feedback so we can make EC2 deployments of Kubernetes as fast and reliable as possible.    
```

@sftim do we have any history of providing Editor's Notes? is this the format they'd be?

Reply:
@nate-double-u We've used editor's notes at the top of other blog posts. I don't like the format of this one, but I'd rather expedite the security concern so we can deliberate formatting in a non-blocking way.